certbot/tests/util_test.py

```diff
@@ -140,9 +140,9 @@ def setUp(self):
         super(MakeOrVerifyDirTest, self).setUp()
 
         self.path = os.path.join(self.tempdir, "foo")
-        os.mkdir(self.path, 0o400)
+        os.mkdir(self.path, 0o600)
```

Comment:
Because on Windows, umask is in practice useless. When you call `os.mkdir`, there is no difference between `0o400` and `0o600`. Folder will be readable and writable for the user who created it. So when you get actual permission, it will be always `0o600`. So instead of making specific code for Linux and Windows, I prefer to use a umask that make sense both for Linux and Windows.

Not the best way indeed, but again ===> #6356.

Same answer for the other questions related to file modes below.